porter/porter.go

```diff
@@ -18,9 +18,13 @@ var (
 	DefaultPorterVersion = "v1.0.0-alpha.19"
```

Comment:
Should we update this default version to rc?

Reply:
oh yeah 🤦‍♀️